### DISCUSSION TITLE
tutorials/intro-typescript/09-querying-real-data

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#125] [course feedback]
# tutorials/intro-typescript/09-querying-real-data

https://www.apollographql.com/tutorials/intro-typescript/09-querying-real-data

**Comment by Alex331349470** [comment]
when I query the data ,i got this error message:"request to https://spotify-demo-api-fe224840a08c.herokuapp.com/v1/browse/featured-playlists failed, reason: getaddrinfo EAI_AGAIN spotify-demo-api-fe224840a08c.herokuapp.com", what's worng with this api data fetching?

**Reply by mabuyo** [reply]
Hey there @Alex331349470 - it looks like that was a timeout error. When I access the [direct link](https://spotify-demo-api-fe224840a08c.herokuapp.com/v1/browse/featured-playlists) now, I do get data back. Could you try again?

**Reply by Alex331349470** [reply]
The reason is I use a proxy, when I use global-agent to set a boostrap, it was fine! tks!

**Reply by mabuyo** [reply]
Ahh, good to hear you got it resolved!
